.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
 <!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
+HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
+AVOID DUPLICATES BY SEARCHING BEFORE CREATING A NEW ISSUE.
```

Comment:
I am proposing to add a paragraph nudging people to posting questions elsewhere. The fact that we accept questions on GitHub does not mean GitHub is the best place where to ask them.

```
HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
AVOID DUPLICATES BY SEARCHING BEFORE CREATING A NEW ISSUE.

If you have a question then please consider using a more suitable venue:
 - https://stackoverflow.com/questions/tagged/loopbackjs
 - https://groups.google.com/forum/#!forum/loopbackjs
 - https://gitter.im/strongloop/loopback
```

Reply:
I was avoiding it because it adds extra text to read at the start of the template BUT I do think it's necessary and most other projects have something along those lines.